script/check_version.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+# So that users can run this script from anywhere and it will work as expected.
+readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+readonly REPO_DIR="$(dirname "$SCRIPT_DIR")"
+
+source "$SCRIPT_DIR/common.sh"
+
+# Sets CHANGED_PACKAGE_LIST and CHANGED_PACKAGES
+check_changed_packages
```

Comment:
Can't we do this in Dart so we're not adding more logic to shell scripts that we need to deal with?